docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/snowflake.md

```diff
@@ -0,0 +1,301 @@
+---
+sidebar_position: 2
+---
+
+# Snowflake Destination
+
+In order to create an end to end test of the ELT pipeline apart from the Source which is the KYVE network, you should
+configure a destination where the `records` will be stored.
+
+This page will guide you through setting up an ELT pipeline using Snowflake to fetch data from
+a [KYVE data pool](https://docs.kyve.network/basics/pools.html) and import it into a `Snowflake` database.
+
+## About Snowflake
+
+The KYVE Data Pipeline enables easy import of KYVE data into any data warehouse or destination
+supported by [Airbyte](https://airbyte.com/). With the [ELT](https://en.wikipedia.org/wiki/Extract,_load,_transform)
+format, data analysts and engineers can now confidently source KYVE data without worrying about its validity or
+reliability.
+
+This tutorial will guide you through setting up an ELT pipeline to fetch data from
+a [KYVE data pool](https://docs.kyve.network/basics/pools.html) and import it into the Snowflake data warehouse.
+
+A data warehouse is a centralized repository of data that is optimized for querying and analysis. It is designed for
+[analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing). A data warehouse can handle
+large volumes of data and complex queries, while a relational database is more suitable for handling small transactions
+in real-time. A data warehouse provides a way for businesses to consolidate data from various sources into a single
+location, which can then be used to make better decisions.
+
+[Snowflake](https://www.snowflake.com/en/) is used by a wide range of businesses and industries, from startups to large
+enterprises. Some main user groups that utilize Snowflake include business analysts, data scientists, data
+engineers and executives.
+
+## Snowflake Setup
+
+### Prerequisites
+
+These instructions assume that you already have a Snowflake account. For more information on how to get started with an
+account, go [here](https://signup.snowflake.com/).
+
+Once the Snowflake data warehouse is set up, you can connect to it in various ways. We are going to be utilizing the UI
```

Comment:
worth highlighting here that the goal is to connect to it from Airbyte? something like this:
```suggestion
Once the Snowflake data warehouse is set up, we can configure it as a destination for Airbyte. We are going to be utilizing the UI
```